### DISCUSSION TITLE
Optionally disable cache flush on app start-up

### DISCUSSION THREAD

**Discussion by maximeclaisse** [directus/directus#15272] [Feature Requests]
When directus runs on a scalable environment, each new instance launched to face traffic or load will flush the cache when it starts.
Cache is very important at this moment, and it's a pity to flush it, as it will generate even more traffic / load on servers.

**Suggestion** : disable cache flush on start-up with env variable (CACHE_FLUSH_ON_STARTUP)

Current src/app.ts
```
export default async function createApp(): Promise<express.Application> {
  ...
  await flushCaches();
  ....
}
```

Suggested src/app.ts
```
export default async function createApp(): Promise<express.Application> {
  ...
  if (env.CACHE_FLUSH_ON_STARTUP === true) {
    await flushCaches();
  }
  ....
}
```

Default value could be true to keep existing behaviour.

**Comment by azrikahar** [comment]
Seems like #18238 ended up implementing this by removing it altogether (but moved into when migrations are run)!

Moving this to approved feature request (retroactively) and marking this as resolved.
